docs/superpowers/plans/2026-05-22-nanosecond-timestamps.md

```
@@ -2338,10 +2341,14 @@ err := row.Scan(&id, &createdAt)
 t := createdAt.Time()
 ```
 
-**Why BIGINT instead of `TIMESTAMPTZ`:** structural byte-equal AAD reconstruction (INV-TS-5), no `Truncate(time.Microsecond)` discipline, deterministic ordering at nanosecond resolution. See the spec for the full rationale and the rejected alternative (`timestamp9` PG extension).
-
-**Enforcement:** `task lint:no-timestamptz` rejects new `TIMESTAMPTZ`/`TIMESTAMP` columns. Escape hatch: `-- pgnanos-exempt: <reason>` on the same line.
-```
+- A `**Why BIGINT instead of `TIMESTAMPTZ`:**` paragraph covering: structural
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)